Dockerfile

```diff
@@ -16,6 +16,10 @@ ENV ANDROID_HOME "/sdk"
 ENV ANDROID_NDK_HOME "/ndk"
 ENV PATH "$PATH:${ANDROID_HOME}/tools"
 
+RUN apt-get update && apt-get install -y locales && rm -rf /var/lib/apt/lists/* \
```

Comment:
you execute apt-get update and throw anything away under /var/lib/apt/lists, only to make an apt-get update again two lines below?

Reply:
Good catch. Fixed.